quarkus-workshop-super-heroes/docs/src/docs/asciidoc/core-appendix/appendix-installing-curl.adoc

```diff
@@ -75,7 +14,13 @@ curl http://localhost:8083/api/heroes
 But what we want is to format the JSON payload, so it is easier to read.
 For that, there is a neat utility tool called `jq` that we could use.
 `jq` is a tool for processing JSON inputs, applying the given filter to its JSON text inputs, and producing the filter's results as JSON on standard output.footnote:[jq https://stedolan.github.io/jq]
+
+== Installing jq
```

Comment:
Can this content move to the main prereqs section? It was moved to the appendix in error (by me), and shouldn't be in the appendix, because it's needed.

Reply:
I've moved jq to the prerequisites and left curl in the appendixe. In the intro I've added: "cURL (should already be installed in your OS, if not, check xref:appendix-installing-curl[the appendix for installation instructions]) and `jq`"